server/code_switch/strategies/goldfish_strategy.py

```diff
@@ -0,0 +1,46 @@
+import random
+
+from code_switch.strategies.cs_strategies import CSStrategy, CSParameters, DEFAULT_CS_PARAMS
+from code_switch.utils import hazard
+
+class GoldfishStrategy(CSStrategy):
+	def __init__(self, cs_params: CSParameters=None):
```

Comment:
the constructor of this class should match the super class

Reply:
I added a parameter called cs_params to the parent class (CSStrategy), although not all strategies require parameters (e.g. English-Only & Spanish Only).